src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,6 +14,8 @@
     private Phone phone;
     private Email email;
     private Address address;
+    private int sequenceNumber;
+    private static int nextSequenceNumber;
```

Comment:
The code would be readable if you assigned the initial value.
BTW, what is the default value assigned when you don't initialize a static variable? Does it make semantic sense to leave it at the default?